portal_convert.go

```diff
@@ -397,8 +396,7 @@ func (puppet *Puppet) addWebhookMeta(part *ConvertedMessage, msg *discordgo.Mess
 		"avatar_mxc": avatarURL.String(),
 	}
 	part.Extra["com.beeper.per_message_profile"] = map[string]any{
-		"is_multiple_users": true,
-
+		"id": msg.Author.ID,
```

Comment:
This is wrong, webhooks don't have IDs. Probably best to use the username as the id (or maybe a hash of username + avatar id)